README.md

```diff
@@ -40,6 +40,14 @@ jobs:
         user-name: 'me@myenv.onmicrosoft.com'
         password-secret: ${{ secrets.MYPASSWORD }}
 
+    - name: Upgrade Solution
```

Comment:
let's discuss offline, but I recommend not adding this to this flow snippet in the readme. Please also remove the Publish Solution from the readme sample
Reason: the flow sample/snippet in the readme is intended to be an illustration, NOT to be exhaustive